src/app/services/data/origin-destination.service.ts

```diff
@@ -27,6 +28,18 @@ export class OriginDestinationService {
     private trainrunService: TrainrunService,
   ) {}
 
+  /**
+   * Returns the origin/destination nodes used for output.
+   *
+   * Note that other nodes may be used for the calculation.
+   */
+  odNodes(): Node[] {
```

Comment:
```suggestion
  getODOutputNodes(): Node[] {
```

Nit: since it is a method, I would prefer using a verb, i.e. `get`